### PR TITLE
Javascript dynamic display

### PR DESCRIPTION
**Pull request by ekenecf**
Created a dynamic pop up window using 
1. Javascript objects for storing information needed
2. Using a for loop to iterate through the arrays,
3. Using inner HTML  to place the targeted

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,39 @@
...
+
+# Prerequisites
+The basic requirements for building this project are:
+
+VSCode or any other equivalent code editor
+Node package manager
+
+# Live Demo:
+
+ekenecf.github.io
```
Comment (Deepakdanger) [inline README.md]: - [ ] Right now this is text, let's change it to link.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @ekenecf, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] The icon got interchange, Kindly keep it as it is as in the original website. 

Screenshot from 2022-03-12 23-20-01

- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ekenecf) [reply README.md]: Hi @Deepakdanger. As requested by you,
1. I have made the changes on the readme file as I have changed the text to a link,
2. I have re-arranged the modal images.

**Review by Deepakdanger — APPROVED** [review]
Hi @ekenecf, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] In case of commit,Please consider:-
- Commit should be in the present tense.
- Commit starts with a Capital letter. For ex:- `Fix linter issue in style.css file`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**